src/runaqa.ts

```diff
@@ -96,7 +102,11 @@ export async function runaqaTest(
 }
 
 function getTestJdkHome(version: string, jdksource: string): string {
-  let javaHome = process.env[`JAVA_HOME_${version}_X64`] as string
+  // Try JAVA_HOME first and then fall back to GITHUB actions default location
+  let javaHome = process.env['JAVA_HOME'] as string
+  if (javaHome === undefined) {
+    javaHome = process.env[`JAVA_HOME_${version}_X64`] as string
```

Comment:
javaHome is the JDK image home we are aiming to test, which depends on the jdksource parameter. It differs from the default JAVA_HOME. So we can't try JAVA_HOME first and fall back to other location.

Reply:
hmm okay, I can switch the logic round